src/transformers/modeling_tf_utils.py

```diff
@@ -610,22 +610,23 @@ def init_copy_embeddings(old_embeddings, new_num_tokens):
     old_num_tokens, old_embedding_dim = shape_list(old_embeddings)
     size_diff = new_num_tokens - old_num_tokens
 
+    # get detached tensor if not already detached
+    old_embeddings = old_embeddings.values() if hasattr(old_embeddings, "values") else old_embeddings
```

Comment:
```suggestion
    old_embeddings = old_embeddings.values() if hasattr(old_embeddings, "values") else old_embeddings
```
It's `value` beneath (without an s). Is the s intended?

Reply:
Good catch! It's `value()` for sure!